README.md

```diff
@@ -17,6 +17,7 @@ including
   - VIC20
   - C16/C116 and Plus/4
   - C64
+  - C64DTV
```

Comment:
I see, there is just new cpu support for DTV but not target. In fact I had wanted to extend only the cpu (option), so I reverted that commit. As a result, README.md is not modified.

Reply:
How about "C64/C64DTV"? BTW a special TGI would be fine.

Reply:
> How about "C64/C64DTV"? BTW a special TGI would be fine.
TGI? Maybe later. I do not know so well the cc65 ecosystem yet. (I'm about to mod my DTV device at first so that I could upload programs.)

Reply:
> I do not know so well the cc65 ecosystem yet.

We follow the usual GitHub model of 'fork & pull request' - see https://gist.github.com/Chaser324/ce0505fbed06b947d962

Apart from that there are some hints on the cc65 code style - see https://github.com/cc65/wiki/wiki#this-cc65-projects-coding-guidelines

And regarding the TGI driver model it's best to take a look at the existing drivers.